packages/gcc_tools.rb

```diff
@@ -17,16 +17,19 @@ def self.build
 
 # define constants
 CREW_PREFIX="${CREW_PREFIX:-/usr/local}"
-ARCH="$(uname -m)"
+ARCH_ACTUAL="$(uname -m)"
+[[ "$ARCH_ACTUAL" == "armv8l" ]] && ARCH_ACTUAL="armv7l"
+ARCH="${ARCH:-$ARCH_ACTUAL}"
 LIB_SUFFIX=""
 [ "${ARCH}" == "x86_64" ] && LIB_SUFFIX="64"
+CURL="LD_LIBRARY_PATH=/usr/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} /usr/bin/curl"
```

Comment:
Is the idea here to bypass the Chromebrew curl?

Reply:
Yes but now that I think of it, it might be best to hide this behind a conditional checking to see if the chromebrew curl is erroring out.

Reply:
I went through a lot of blood, sweat and tears to make sure the Chromebrew curl doesn't error out. :)

Reply:
Yes but what about when I muck about with gcc and suddenly curl can't find libraries?

(This literally happened to me yesterday.)

Reply:
You are not supposed to be mucking around with gcc but testing on fresh installs. :)

Reply:
mucking around meaning testing gcc builds built on fresh installs on my own install to see if anything breaks.

Have to dogfood sometimes!

Reply:
Made changes to the curl logic to see if curl works, and took out the ARCH complications. Is this more bearable?